src/attributes.ts

```diff
@@ -644,6 +644,19 @@ const ATTRIBUTES_DEFINITION_BASE = {
 		name: 'A chainable armbinder',
 		description: 'An armbinder that is chainable to another item',
 	},
+	// Jute Rope
+	Waist_rope_anchor_point: {
+		name: 'Rope around waist',
+		description: 'A rope around the waist',
+	},
+	Vulva_rope_anchor_point: {
+		name: 'Crotch rope link',
+		description: 'A rope linking crotch rope to tied hands',
+	},
```

Comment:
Suggested wording that I feel better matches the current usage:
Name: Crotch rope
Description: A crotch rope that a linking rope can be tied to


Context in which it is used currently (when trying to select the option requiring this, not yet having it selected):
> The arm jute ropes requires "A rope linking crotch rope to tied hands" (must be worn under the arm jute ropes).

After suggestion:
> The arm jute ropes requires "A crotch rope that a linking rope can be tied to" (must be worn under the arm jute ropes).